src/component/variantPage/TherapeuticImplication.tsx

```diff
@@ -1,11 +1,11 @@
 import * as React from 'react';
 import { observer } from 'mobx-react';
 import _ from 'lodash';
+import { IndicatorQueryTreatment } from 'cbioportal-frontend-commons/dist/api/generated/OncoKbAPI';
```

Comment:
ideally, we should expose `IndicatorQueryTreatment` in cbioportal-frontend-commons, so that you can just do

```
import { IndicatorQueryTreatment } from 'cbioportal-frontend-commons'
```

But, even more ideally OncoKbAPI should be a separate package in the cbioportal-frontend monorepo, so that you can simply do

```
import { IndicatorQueryTreatment } from 'oncokb-api-client';
```

Reply:
@onursumer that would be great, but for now I guess I still need to use `import { IndicatorQueryTreatment } from 'cbioportal-frontend-commons/dist/api/generated/OncoKbAPI';` right?

Reply:
right, but maybe mark these kind of imports so that we don't forget to update them later

Reply:
@onursumer thanks, i add a TODO there